### PR TITLE
Add me as maintainer

### PR DESCRIPTION
**Pull request by domoritz**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -36,3 +36,4 @@ extra:
   recipe-maintainers:
     - jakevdp
     - ocefpaf
+    - domoritz
```
Comment (ocefpaf) [inline recipe/meta.yaml:39]: Thanks @domorits! Can you confirm that the home url moved to `https://github.com/vega/ipyvega`? We had `http://github.com/vega/ipyvega/tree/vega3` before.

(If so we can update that info in this PR.)

**Comment by domoritz** [comment]
Yes but I also deprecated the `vega3` package in favor of the `vega` package. I don't know what the right mechanism for deprecating is so I just added a warning to the package code.

**Comment by ocefpaf** [comment]
> Yes but I also deprecated the `vega3 `package in favor of the `vega` package. I don't know what the right mechanism for deprecating is so I just added a warning to the package code.

I'm confused. I though that `vega3` was the sucessor of the `vega` package we had in https://github.com/conda-forge/vega-feedstock/

If that is not the case we should deprecate this feedstock and go back to https://github.com/conda-forge/vega-feedstock/

**Comment by ocefpaf** [comment]
@domoritz I saw that you update the `vega` feedstock recently in https://github.com/conda-forge/vega-feedstock/pull/21

I will merge this PR but I guess that the right thing to do is to archive this feedstock.

Ping @jakevdp who is planning to use `vega3` in his tutorial. I guess that we can now use `vega` :smile:

**Comment by jakevdp** [comment]
Can we wait until the next Altair release to archive this? The Altair documentation still recommends installing ``vega3``.

**Comment by domoritz** [comment]
I'm okay with waiting. I pushed a package to pypi that shows a warning, though.

screen shot 2018-05-08 at 15 14 51
(in `__init__.py`)

**Comment by ocefpaf** [comment]
> Can we wait until the next Altair release to archive this?

Sure, but archiving this does not mean we remove the existing packages. Archiving is only to reduce the maintenance burden. However, since there will be a new release with the deprecation, let's wait for that.

**Pull request merged**